demos/python_demos/multi_camera_multi_person_tracking/tools/run_evaluate.py

```diff
@@ -0,0 +1,153 @@
+"""
+ Copyright (c) 2019 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import argparse
+import json
+import logging as log
+
+import motmetrics as mm
+import numpy as np
+from lxml import etree
+from tqdm import tqdm
+
+from mc_tracker.sct import TrackedObj
```

Comment:
This import works only if current directory is `open_model_zoo/demos/python_demos/multi_camera_multi_person_tracking`

Reply:
Fixed. My fix probably is not so pretty but it resolves the problem.

Reply:
Updating `sys.path` is usually considered as a hack. I suggest moving the tools up one folder to stay at the level of the demo.

Reply:
OK, moved to parent directory.